mev_inspect/arbitrages.py

```diff
@@ -92,11 +92,21 @@ def _get_shortest_route(
     start_swap: Swap,
     end_swaps: List[Swap],
     all_swaps: List[Swap],
+    max_route_length: Optional[int] = None,
 ) -> Optional[List[Swap]]:
+    if len(end_swaps) == 0:
+        return None
+
+    if max_route_length is not None and max_route_length < 2:
+        return None
+
     for end_swap in end_swaps:
         if start_swap.token_out_address == end_swap.token_in_address:
             return [start_swap, end_swap]
 
+    if max_route_length is not None and max_route_length == 2:
```

Comment:
Is there a reason to separate this line from:

``` if max_route_length is not None and max_route_length < 2:```

why not:

``` if max_route_length is not None and max_route_length <= 2``` ?

Reply:
because if it were less than 2 it'd have already cut out on line 100

Reply:
Right, that's my point; is there a reason to not include them in the same line?

Reply:
yes because the line above it can return the case of len 2, but nowhere below can